modules/exploits/multi/handler.rb

```diff
@@ -53,6 +53,11 @@ def initialize(info = {})
   end
 
   def exploit
+    if datastore['PAYLOAD'].starts_with? "cmd/"
```

Comment:
I think checking that the architecture is `ARCH_CMD` would be better here. It'd be more future proof at least.